sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLPoolDAO.java

```diff
@@ -526,18 +517,8 @@ public String mapRow(ResultSet rs, int pos) throws SQLException {
         esParams.addValue("elementType", d.getClass().getName());
 
         eInsert.execute(esParams);
+        // TODO Fixed in Hibernate
```

Comment:
Please leave original code commented out with a note that describes the necessary change. Code should not be removed entirely until class is Hibernatized to ensure that person doing that work understands the requirements and can verify they are met (and so reviewers can easier see the behaviour moved and unchanged)